ClosedXML/Excel/XLWorksheet.cs

```diff
@@ -1350,35 +1336,57 @@ private void MoveNamedRangesColumns(XLRange range, int columnsShifted, IXLNamedR
 
         public void NotifyRangeShiftedRows(XLRange range, Int32 rowsShifted)
         {
-            if (RangeShiftedRows != null)
+            try
```

Comment:
Why inside a `try`?

Reply:
For a single failed `shift` would not leave events for entire sheet turned off. Exception is not swallowed though.

Reply:
Oh, I see... no `catch` block.